### PR TITLE
MAT-7475: remove unneeded feature flags

### PR DESCRIPTION
**Pull request by ethankaplan**
## MADiE PR

Jira Ticket: [MAT-7475](https://jira.cms.gov/browse/MAT-7475)
(Optional) Related Tickets:

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included.
* [ ] No extemporaneous files are included (i.e Complied files or testing results).
* [ ] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/Store/featureFlagStore.ts [hunk 1]
```diff
@@ -5,15 +5,11 @@ import { BehaviorSubject } from "rxjs";
 const subject = new BehaviorSubject<FeatureFlags | null>(null);
 export interface FeatureFlags {
   CQLBuilderIncludes: boolean;
-  includeSDEValues: boolean;
   qdmExport: boolean;
```
Comment (adongare) [inline src/Store/featureFlagStore.ts]: I think one needs to be removed as well

File: src/Store/featureFlagStore.ts [hunk 2]
```diff
@@ -24,15 +20,11 @@ export interface FeatureFlags {
 }
 const initialState: FeatureFlags = {
   CQLBuilderIncludes: false,
-  includeSDEValues: false,
   qdmExport: false,
```
Comment (adongare) [inline src/Store/featureFlagStore.ts]: and this

**Review by adongare — APPROVED** [review]
